src/utils.js

```diff
@@ -176,3 +176,25 @@ export function getNativeValueFromMarkupArgumentType(value, type) {
   }
   return value
 }
+
+// calculate the number of addresses according to the amount of wits passed
+export function calculateAddressesNumber(amount) {
+  if (amount === 0) return 0
+  const ceil50 = x => x + 50 - (x % 50)
```

Comment:
Ceil(50) should return 50, not 100

```suggestion
  const ceil50 = x => Math.ceil(x / 50) * 50
```